modules/agar/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Unreleased
 
+### Fixed
+- Creating native events using DragEvent or ClipboardEvent constructors using agar DataTransfer instances would fail on Firefo. #TINY-10386
```

Comment:
```suggestion
- Creating native events using DragEvent or ClipboardEvent constructors using agar DataTransfer instances would fail on Firefox. #TINY-10386
```